.bingo/prometheus.mod

```diff
@@ -20,4 +20,43 @@ replace (
 	k8s.io/klog => github.com/simonpasquier/klog-gokit v0.1.0
 )
 
-require github.com/prometheus/prometheus v2.4.3+incompatible // cmd/prometheus
+require (
```

Comment:
It's better to remove unrelated changes. (: This got changed and it's fine, but it can cause problems with future reverts if we keep unrelated changes in one PR.

Reply:
yes right! removed them